src/main/kotlin/org/dashevo/dapiclient/OldDapiClient.kt

```diff
@@ -0,0 +1,254 @@
+/**
```

Comment:
Do we still need this file?

Reply:
We don't need it.  It is always in the git history if it is required later for reference.

Reply:
Removed here: b8f87e4